### ISSUE TITLE
Will you update it to 1.21 (eventually)?

### ISSUE THREAD

**Issue by Meteoblastdragon** [PieKing1215/DripSounds#20]
Not to rush or anything, but is there a plan to update this mod to 1.21 and if so, do you have an ETA?

It's not meant to rush you but more so for me to know if I should keep an eye out on this mod.

**Comment by vico93** [comment]
Waiting for this as well!

**Comment by PieKing1215** [comment]
Hey! Yeah this will be updated eventually, no promises how soon though
I want to do some reorganization/cleanup to make it easier to maintain in the future and I might do that before updating to 1.20+

**Comment by Meteoblastdragon** [comment]
Cool! Take your time and good luck! Now I know to keep this in my modpack and just keep it disabled until there's an update!

**Comment by SussyBob** [comment]
probably archieved, still waiting for next update tho..

**Comment by PieKing1215** [comment]
There's a couple automation things I need to finish before releasing but if anyone wants to try out dev builds of v0.4 you can access them here (NOTE: dev builds are not well-tested)
https://github.com/PieKing1215/DripSounds/actions?query=branch%3Astonecutter+is%3Asuccess

Sign into GitHub, click the top one, scroll down to artifacts and click the file to download
Inside the zip there's jars for various Minecraft versions from 1.17 to 1.21.4 for Fabric/NeoForge/Forge

If anyone tries it lmk if you have any issues

**Comment by LordEnder-Kitty** [comment]
I don't see the files or anything on the link.

EDIT: I just downloaded the branch and compiled it myself. Seems to all be working well in game.

**Comment by PieKing1215** [comment]
Alright just released [v0.4.0](https://github.com/PieKing1215/DripSounds/releases/tag/v0.4.0) with support for 1.17 to 1.21.4 ([also released on Modrinth](https://modrinth.com/mod/dripsounds/versions))

If anyone runs into problems feel free to open another issue :)
